packages/dataviews/src/style.scss

```diff
@@ -1,12 +1,12 @@
 .dataviews-wrapper {
-	width: 100%;
```

Comment:
This style makes the assumption that the parent div doesn't have a padding which is not always true.